test/helpers/helpers.js

```diff
@@ -18,23 +15,8 @@ beforeEach(() => {
           if (result.pass) {
             result.message = spec.markdown + '\n------\n\nExpected: Should Fail';
           } else {
-            var expectedHtml = expected.replace(/\s/g, '');
-            var actualHtml = actual.replace(/\s/g, '');
-
-            for (var i = 0; i < expectedHtml.length; i++) {
-              if (actualHtml[i] !== expectedHtml[i]) {
-                actualHtml = actualHtml.substring(
-                  Math.max(i - EXCERPT_LENGTH, 0),
-                  Math.min(i + EXCERPT_LENGTH, actualHtml.length));
-
-                expectedHtml = expectedHtml.substring(
-                  Math.max(i - EXCERPT_LENGTH, 0),
-                  Math.min(i + EXCERPT_LENGTH, expectedHtml.length));
-
-                break;
-              }
-            }
-            result.message = 'Expected:\n' + expectedHtml + '\n\nActual:\n' + actualHtml;
+            const diff = htmlDiffer.firstDiff(actual, expected);
```

Comment:
```suggestion
            const { suggested, actual } = htmlDiffer.firstDiff(actual, expected);
```

Reply:
destructuring is not available in node 4